pom.xml

```diff
@@ -105,26 +105,29 @@
 		<!-- Quarkus Version -->
 		<quarkus.version>1.3.1.Final</quarkus.version>
 
-		<!-- Jandex -->
-		<jandex.version>1.0.7</jandex.version>
+        <!-- Schema types -->
```

Comment:
Duplicated `<!-- Schema types -->`

Reply:
Thanks @jsenko I must of missed it when resolving merge conflicts